mix.exs

```diff
@@ -93,7 +95,8 @@ defmodule Phoenix.MixProject do
       {:mint_web_socket, "~> 1.0.0", only: :test},
 
       # Dev dependencies
-      {:esbuild, "~> 0.5", only: :dev}
+      {:esbuild, "~> 0.5", only: :dev},
+      {:castore, ">= 0.0.0", only: [:dev, :test]}
```

Comment:
We will either need it as a dependency OR add it to the generated mix.exs files. I will add it as a dep. We can drop it on OTP 25+ as Erlang has its own certs now.